Block/Adminhtml/Header.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * The Container class.
+ *
+ * This handles the Magento data and allows it to be retrieved by the view templates.
+ */
+
+declare(strict_types=1);
+
+namespace NS8\Protect\Block\Adminhtml;
+
+use Magento\Backend\Block\Template;
+use Magento\Backend\Block\Template\Context;
+use NS8\Protect\Helper\Config;
+
+/**
+ * The Header class.
+ *
+ * This handles the Magento data and allows it to be retrieved by the view templates.
+ */
+class Header extends Template
+{
+    /** @var Config */
+    public $config;
+
+    /**
+     * Constructor
+     *
+     * @param Context $context The context
+     */
+    public function __construct(
+        Config $config,
+        Context $context
+    ) {
+        parent::__construct($context);
+
+        $this->config = $config;
+    }
+
+    public function getStores(): array
```

Comment:
Please add a docblock to all public methods.

Reply:
meant to delete that entire class 👍

Reply:
Guess that works too. 😸